rero_ils/modules/loans/api.py

```diff
@@ -706,70 +706,129 @@ def is_notified(self, notification_type=None, counter=0):
         return number_of_reminders_sent(
             self, notification_type=notification_type) > counter
 
-    def create_notification(self, notification_type=None, counter=0):
+    def get_notification_candidates(self, trigger):
+        """Get notifications candidate to create for the notification.
+
+        This function will check the loan and return all possible notifications
+        to create related to it. In some case, a notification must be generate
+        on an other loan than itself (in case of request).
+
+        :param trigger: the fired action trigger (optional)
+        :return a list of tuple. Each tuple represent a notification candidate.
+                Each tuple is composed by 2 elements : the loan object, and the
+                related notification type.
```

Comment:
```suggestion
        :return a list of tuple. Each tuple represent a candidate notification.
                Each tuple is composed by 2 elements : the loan object, and the
                related notification type.
```

Reply:
this is a candidate to be a notification if future. In my sense, it's better as 'notification candidate'